common/run/v1alpha/run.proto

```diff
@@ -25,3 +28,26 @@ enum RunSource {
   // Run from API or SDK.
   RUN_SOURCE_API = 2;
 }
+
+// TimeRangeType defines the time range of a trend.
+enum TimeRangeType {
+  // Unspecified.
+  TIME_RANGE_TYPE_UNSPECIFIED = 0;
+  // Today.
+  TIME_RANGE_TYPE_TODAY = 1;
+  // Last 24 hours.
+  TIME_RANGE_TYPE_LAST_24_HOURS = 2;
+  // Last 7 days.
+  TIME_RANGE_TYPE_LAST_7_DAYS = 3;
+  // Last 30 days.
+  TIME_RANGE_TYPE_LAST_30_DAYS = 4;
+}
+
+// CreditTrendData represents the trend data of credits with timestamp.
```

Comment:
The documentation is a bit unclear here. Could you specify what `trend` means? I understand is the difference between the previous period and the current one? Is it absolute or relative? If so, I wouldn't have a specific resource for this. Clients can fetch the data from the current and the previous period and compute the trend on their own.

Also, the `Data` suffix is innecessary, it doesn't add any extra information.

Reply:
This is not required anymore. Will use the rpc and msg in your previous pull requests for metrics API instead